### PR TITLE
Upgrade storybook and other packages, and adjust linting rules

### PR DESCRIPTION
**Pull request by haraldvinje**
- **:pushpin: Update node version**
- **:pushpin: Update storybook and vite**
- **:pushpin: General minor and patch updates**
- **:rotating_light: Update linter rules**

### CODE REVIEW

**Comment by haraldvinje** [comment]
Screenshot from 2024-03-20 11-59-28
[Object object] problem is resolved in this version!

**Review by henrikhermansen — APPROVED** [review]

File: packages/react/src/skeleton/skeleton.stories.tsx [hunk 1]
```diff
@@ -130,26 +130,26 @@ export const Article: Story = {
...
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(25, 60)} />
+        <Skeleton variant="text" width={numberBetween(45, 80)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
```
Comment (juliuxu) [inline packages/react/src/skeleton/skeleton.stories.tsx:152]: In this example I wanted the width to be actual percentage of the full width. Maybe the api needs a little bit of tinkering 🤔 

You see here the style is just passed down directly: https://github.com/bring/hedwig-design-system/blob/master/packages/react/src/skeleton/skeleton.tsx#L54-L62

Wouldn't this change make it so the widths become something between 75px and 90px?

**Review by juliuxu — DISMISSED** [review]
🎉 Good stuff 💯  
Great that they fixed the `[Object]` stuff 🙌 

Have a comment about the skeleton story.

File: packages/react/src/skeleton/skeleton.stories.tsx (hunk 1 above)
Comment (juliuxu) [reply packages/react/src/skeleton/skeleton.stories.tsx:152]: 💭 Could make the function be called `percentageBetween(min, max)` instead 🤔

Comment (haraldvinje) [reply packages/react/src/skeleton/skeleton.stories.tsx:152]: Oh, my bad! Didn't notice the % sign! Will change

**Review by juliuxu — APPROVED** [review]
🥳

**Pull request merged**